### ISSUE TITLE
Make widget error state clickable

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nelenkov/wwwjdic#40]
```
To easily recover from network errors and the like
```

Original issue reported on code.google.com by `nikolay....@gmail.com` on 15 Apr 2011 at 1:48

**Comment by GoogleCodeExporter** [comment]
```
done in 1.7.3
```

Original comment by `nikolay....@gmail.com` on 6 May 2011 at 2:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fix didn't quite work: setting an intent on an invisible view -- not a good 
idea...
```

Original comment by `nikolay....@gmail.com` on 19 May 2011 at 3:06
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.8.2
```

Original comment by `nikolay....@gmail.com` on 26 May 2011 at 6:52
- Changed state: **Fixed**
